docs/DataWorkflows/DataCycle.md

```diff
@@ -0,0 +1,671 @@
+[//]: #
+
+See below for examples of cross sections of a cubids-group csv:
+
+[Pre-apply](https://github.com/PennLINC/RBC/tree/master/PennLINC/HRC_BIDS_Fix/HRC_pre_apply)
+[Post-apply](https://github.com/PennLINC/RBC/tree/master/PennLINC/HRC_BIDS_Fix/HRC_post_apply)
+
+Once the edited CSV is ready, the merges, renamings and deletions can be
+applied to your BIDS data. Again from `working/`, if you're not using
+datalad at this point, run
+
+```shell
+$ cubids-apply \
+    BIDS \
+    code/iterations/iter0_summary.csv \
+    code/iterations/iter0_files.csv \
+    code/iterations/apply1
+```
+
+If using datalad, be sure to include `--use-datalad` as the first argument to
+`cubids-apply`. If not using Datalad, be sure to add to your *Data Narrative*
+so you know that CuBIDS was used to change your data. Describe any major
+renaming, deleting or merging of parameter groups.
+
+New parameter groups will be described in `code/iterations/apply1*.csv` and you
+should make sure your data was changed as intended.
+
+Once you are satisfied with your key/parameter groups, be sure to re-run
+`cubids-validate` to make sure you haven't introduced any BIDS-incompatible
+names.
+
+Congratulations! You have created a fully-curated BIDS dataset.
+
+This data is typically stored on PMACS as BIDS datasets, transferred from CUBIC after curation via `rsync`. 
+
+
+
+# Getting your data from PMACS to CUBIC
+Data can be fetched back from PMACS, if needed, using the documentation specified [here](/docs/DataWorkflows/FetchingYourData.md). At the moment, all the data we have is already on PMACS and you will not need to curate any legacy data/ data that has already been collected a while back. These datasets and the links to fetch them are listed [here](./AvailableStaticData.md).
+
+# Unzipping your data
+
+Occasionally, data may be zipped. More information about this can be found [here](/PennLINC.github.io/docs/DataWorkflows/FetchingYourData.md)!
+
+# Processing your data
+[BABS](https://pennlinc-babs.readthedocs.io/en/latest/) is a quick and easy tool for processing pipelines via [Datalad](./Datalad.md). The documentation is thorough, with information on setting up, installing, and running BABS on data. An example walkthrough is also available via the documentation. 
+
+# Sharing your data
+{: .no_toc}
+
+Collaboration is an important part of our work. Sharing the input data and outputs/derivatives of a project is a critical step in the scientific process, and so doing it both accurately and efficiently is a high priority. Overall, remember that this step can be a strain, especially when dealing with external collaborators or systems that are foreign to you. Be patient, and remember that we have to balance accuracy/reproducibility with efficiency/speed.
+
+Below are a few example scenarios and what best recommendations we have for sharing data, particularly outputs from BABS
+
+## Datalad to Datalad
```

Comment:
Can you add a section on Datalad to OpenNeuro? See #60. Can be left empty here, but will remind us to fill it out at some point.

Reply:
added the section + linked to docs on how to do it